Vladyslav_Gurevych/HW8/Map.py

```diff
@@ -0,0 +1,36 @@
+def own_map(callback, data: list):
```

Comment:
annotation for a callback. 

from typing import Callable

callback: Callable

Reply:
added in all tasks